cmd/internal/kube/secret.go

```diff
@@ -0,0 +1,127 @@
+package kube
+
+import (
+	"encoding/base64"
+	"log"
+	"os"
+	"path/filepath"
+	"text/template"
+
+	"github.com/akitasoftware/akita-cli/printer"
+
+	"github.com/akitasoftware/akita-cli/cmd/internal/cmderr"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	output    string
+	namespace string
+	// Store a parsed representation of /template/akita-secret.tmpl
+	secretTemplate *template.Template
+)
+
+var secretCmd = &cobra.Command{
+	Use:   "secret",
+	Short: "Generate a Kubernetes secret config for Akita",
```

Comment:
I'm not sure 'Kubernetes secret config' is clear?

```suggestion
	Short: "Generate a Kubernetes secret containing the Akita credentials",
```